.eslintrc.js

```diff
@@ -1,275 +1,29 @@
-    "no-path-concat": "error",
-    "no-plusplus": "off",
-    "no-process-env": "off",
-    "no-process-exit": "error",
-    "no-proto": "error",
-    "no-prototype-builtins": "error",
-    "no-restricted-globals": "error",
-    "no-restricted-imports": "error",
-    "no-restricted-modules": "error",
-    "no-restricted-syntax": "error",
-    "no-return-assign": "error",
-    "no-script-url": "error",
-    "no-self-compare": "error",
-    "no-sequences": "error",
-    "no-shadow": "off",
-    "no-shadow-restricted-names": "error",
-    "no-spaced-func": "error",
-    "no-sync": "error",
-    "no-ternary": "off",
-    "no-throw-literal": "error",
-    "no-undef-init": "error",
-    "no-undefined": "off",
-    "no-underscore-dangle": "off",
-    "no-unmodified-loop-condition": "error",
-    "no-unneeded-ternary": "error",
-    "no-unused-expressions": "off",
-    "no-unused-vars": "warn",
-    "no-use-before-define": "off",
-    "no-useless-call": "error",
-    "no-useless-computed-key": "error",
-    "no-useless-concat": "error",
-    "no-useless-constructor": "error",
-    "no-useless-escape": "off",
-    "no-useless-rename": "error",
-    "no-var": "error",
-    "no-void": "error",
-    "no-warning-comments": "off",
-    "no-whitespace-before-property": "error",
-    "no-with": "error",
-    "object-curly-spacing": [
-      "warn",
-      "always"
-    ],
-    "object-property-newline": [
-      "error",
-      {
-        "allowMultiplePropertiesPerLine": true
-      }
-    ],
-    "one-var": "off",
-    "one-var-declaration-per-line": "error",
-    "operator-assignment": "error",
-    "operator-linebreak": "error",
-    "padded-blocks": "off",
-    "prefer-arrow-callback": "off",
-    "prefer-const": "off",
-    "prefer-reflect": "off",
-    "prefer-rest-params": "error",
-    "prefer-spread": "error",
-    "prefer-template": "off",
-    "quote-props": "off",
-    "quotes": [
-      "error",
-      "single"
-    ],
-    "no-this-before-super": "off",
-    "radix": "error",
-    "rest-spread-spacing": "error",
-    "semi": ["warn", "always"],
-    "semi-spacing": "error",
-    "sort-imports": "off",
-    "sort-vars": "error",
-    "space-before-blocks": "error",
-    "space-before-function-paren": "off",
-    "space-in-parens": [
-      "error",
-      "never"
-    ],
-    "space-infix-ops": "error",
-    "space-unary-ops": "error",
-    "spaced-comment": [
-      "error",
-      "always"
-    ],
-    "strict": "error",
-    "template-curly-spacing": [
-      "error",
-      "never"
-    ],
-    "unicode-bom": [
-      "error",
-      "never"
-    ],
-    "valid-jsdoc": "off",
-    "vars-on-top": "error",
-    "wrap-iife": "error",
-    "wrap-regex": "off",
-    "yield-star-spacing": "error",
-    "yoda": [
-      "error",
-      "never"
-    ]
+  rules: {
+    indent: ['error', 2, { SwitchCase: 1 }],
+    'linebreak-style': ['error', 'unix'],
+    quotes: ['error', 'single'],
+    semi: ['warn', 'always'],
+    'no-console': 'off',
```

Comment:
I would at least leave a warning here.

Reply:
I was trying to simplify without causing any change, that's the way it was before.
We can open this for discussion, but changes here will require adjustments to the code

Reply:
I agree it is a different scope, I think that in general you want to forbid the usage of console.log to ensure that debugging statements are not left by mistake, if you agree with me, you can just create a task for later and put at the top of the backlog.

Reply:
I completely agree, in fact my current task is wrapping console.logs with a debug-mode condition :) 
But opening a discussion around such decisions in the video-player context is going to be a huge overhead. I'd say we should have a company level eslint standard (if we don't already) and discuss it at that level.